src/main/resources/config/application-dev.yml

```diff
@@ -136,6 +136,11 @@ artemis:
         destination: https://telemetry.artemis.cit.tum.de
     version-control:
         local-vcs-repo-path: ./local-vcs-repos
+    security:
```

Comment:
I guess we should move this to the application.yml to make sure the default config is valid for all installations. What do you think?